rest/route/user.go

```diff
@@ -90,6 +91,15 @@ func (h *userSettingsPostHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "Error saving user settings"))
 	}
 
+	if h.settings.SpruceFeedback != nil {
+		h.settings.SpruceFeedback.SubmittedAt = time.Now()
+		h.settings.SpruceFeedback.User = model.ToAPIString(u.Username())
+		f, _ := h.settings.SpruceFeedback.ToService()
+		if err = h.sc.SubmitFeedback(f.(dbModel.FeedbackSubmission)); err != nil {
+			return gimlet.MakeJSONErrorResponder(err)
```

Comment:
Wrap this error

Reply:
This is intentionally not wrapped because the error that's returned is specific enough